runtime/makelib/mkconstants.mk.ftl

```diff
@@ -157,6 +157,8 @@ endif
 TR_HOST=TR_HOST_X86
 <#elseif uma.spec.processor.arm>
 TR_HOST=TR_HOST_ARM
+<#elseif uma.spec.processor.aarch64>
+TR_HOST=TR_HOST_ARM64
```

Comment:
Perhaps this should be `TR_HOST=TR_HOST_AARCH64` for consistency?

Reply:
OMR code already uses `TR_HOST_ARM64`.

Reply:
Yes, I see it now. I think that's an unfortunate situation, but not one to be addressed in this PR.